backend/src/dog-walk-day/dog-walk-day.service.ts

```diff
@@ -17,6 +17,11 @@ export class DogWalkDayService {
 
     async getWalkDayList(walkDayIds: number[]): Promise<number[][]> {
         const foundDays = await this.dogWalkDayRepository.find({ where: { id: In(walkDayIds) } });
+
+        if (!walkDayIds.length) {
+            this.logger.error(`walkDayIds에 값이 없습니다`, '');
+            throw new NotFoundException(`walkDayIds에 값이 없습니다`);
+        }
```

Comment:
walkDayIds 유효성 체크는 find 전에 하는 것이 좋을 것 같아요.
```suggestion
        if (!walkDayIds.length) {
            this.logger.error(`walkDayIds에 값이 없습니다`, '');
            throw new NotFoundException(`walkDayIds에 값이 없습니다`);
        }
        
        const foundDays = await this.dogWalkDayRepository.find({ where: { id: In(walkDayIds) } });
```

Reply:
헉 왜 이렇게 했지ㅋㅋ 고칠게요